### PR TITLE
Add support for Rails 8.0

### PR DESCRIPTION
**Pull request by leoarnold**

### CODE REVIEW

**Comment by leoarnold** [comment]
This PR is a **draft** because it is waiting for https://github.com/cgriego/active_attr/pull/205 to be released.

File: strip_attributes.gemspec
Comment (rmm5t) [inline strip_attributes.gemspec:27]: [QUESTION] Please help me understand why this active_attr restriction was necessary as compared to `~> 0.15` (considering the latest release of active_attr is 0.17).

**Review by rmm5t — APPROVED** [review]

File: gemfiles/activemodel-8.0.gemfile
Comment (rmm5t) [inline gemfiles/activemodel-8.0.gemfile:4]: I'm inclined to merge this PR even though the test suite relies on your private branch at the moment. Would you mind coming back to update this after your active_attr PR is merged?

Comment (leoarnold) [reply gemfiles/activemodel-8.0.gemfile:4]: I marked this PR as a **draft** specifically to prevent premature merging. The reason I put up the draft is so that other developers can see that this effort is already in progress and they don't need to investigate themselves (or open issues).

File: strip_attributes.gemspec
Comment (leoarnold) [reply strip_attributes.gemspec:27]: Because we need https://github.com/cgriego/active_attr/pull/205 which will (probably) be published as `active_attr` v0.18.0.

Comment (rmm5t) [reply strip_attributes.gemspec:27]: That doesn't explain the difference between `"~> 0.15"` and `">= 0.15.0", "< 0.19"`

[PLEASE REVERT] back to `~> 0.15`

File: gemfiles/activemodel-8.0.gemfile
Comment (rmm5t) [reply gemfiles/activemodel-8.0.gemfile:4]: Why would a _"premature"_ merge be a problem? This gem doesn't have a required runtime dependency on active_attr.  ActiveAttr is an optional dependency. 

We should be able to merge this PR and when active_attr is updated, we'll get it's benefits for free.

Comment (rmm5t) [reply gemfiles/activemodel-8.0.gemfile:4]: i.e. Keeping this PR as a draft just holds back those projects that depend on strip_attributes but don't have a dependency on active_attr.

**Pull request closed without merging**
